ui/package.json

```diff
@@ -108,6 +110,7 @@
     "@types/yup": "^0.29.14",
     "@wojtekmaj/enzyme-adapter-react-17": "^0.8.0",
     "axios-mock-adapter": "^1.15.0",
+    "bundlemon": "^2.0.0-rc.1",
```

Comment:
@karolina-siemieniuk-morawska this is not the final release, do we need the 2.0.0 features?